simple/stats/config.py

```diff
@@ -49,6 +49,7 @@
 _ROW_ENTITY_TYPE_FIELD = "rowEntityType"
 _ENTITY_COLUMNS = "entityColumns"
 _ENTITIES_FIELD = "entities"
+_GENERATE_HIERARCHY = "groupStatVarByProperty"
```

Comment:
nit change to GROUP_STAT_VAR_BY_HIERARCHY

Reply:
Renamed to `_GROUP_STAT_VARS_BY_PROPERTY`. Also pluralized the property name. i.e. `groupStatVarByProperty` -> `groupStatVarsByProperty`.